core/src/main/resources/hudson/model/View/sidepanel_lt.properties

```diff
@@ -1,29 +1,9 @@
-# The MIT License
-#
-# Copyright (c) 2004-2010, Sun Microsystems, Inc.
```

Comment:
Ran over other translation files and noticed that those varies from file to file. Have a plan to change those in all (*_lt.properties) files to same. But there is a question which is valid and up-to-date?